### PR TITLE
Add title and descriptions from `package.json` to `package.nls.json`

### PR DESCRIPTION
**Pull request by karthiknadig**

### CODE REVIEW

File: package.nls.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
+{
+    "python-envs.defaultEnvManager.description": "The default environment manager to use for creating and managing environments.",
```
Comment (cwebster-99) [inline package.nls.json]:
```suggestion
    "python-envs.defaultEnvManager.description": "The default environment manager for creating and managing environments.",
```

File: package.nls.json [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    "python-envs.defaultPackageManager.description": "The default package manager to use for installing packages in environments.",
```
Comment (cwebster-99) [inline package.nls.json]:
```suggestion
    "python-envs.defaultPackageManager.description": "The default package manager for installing packages in environments.",
```

File: package.nls.json [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+    "python-envs.pythonProjects.description": "The list of Python projects.",
```
Comment (cwebster-99) [inline package.nls.json:4]: Do you think we should provide a brief description of what we consider a "project" to be? Something like A Python project can be a file or folder that contains Python code."

File: package.nls.json [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+    "python-envs.pythonProjects.path.description": "The path to a folder or file in the workspace to treat as Python project.",
+    "python-envs.pythonProjects.envManager.description": "The environment manager to use for creating and managing environments for this project.",
```
Comment (cwebster-99) [inline package.nls.json]:
```suggestion
    "python-envs.pythonProjects.envManager.description": "The environment manager for creating and managing environments for this project.",
```

File: package.nls.json [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+    "python-envs.pythonProjects.packageManager.description": "The package manager to use for managing packages in environments for this project.",
```
Comment (cwebster-99) [inline package.nls.json]:
```suggestion
    "python-envs.pythonProjects.packageManager.description": "The package manager for managing packages in environments for this project.",
```

File: package.nls.json [hunk 6]
```diff
@@ -0,0 +1,30 @@
+{
+    "python-envs.defaultEnvManager.description": "The default environment manager to use for creating and managing environments.",
+    "python-envs.defaultPackageManager.description": "The default package manager to use for installing packages in environments.",
+    "python-envs.pythonProjects.description": "The list of Python projects.",
+    "python-envs.pythonProjects.path.description": "The path to a folder or file in the workspace to treat as Python project.",
```
Comment (cwebster-99) [inline package.nls.json]:
```suggestion
    "python-envs.pythonProjects.path.description": "The path to a folder or file in the workspace to be treated as a Python project.",
```

File: package.nls.json (hunk 3 above)
Comment (karthiknadig) [reply package.nls.json:4]: We can probably also provide a link to wiki where we describe what a project is.

**Pull request merged**